src/__tests__/json-schema.test.js

```diff
@@ -0,0 +1,314 @@
+describe('json-schema', () => {
+  const Sequelize = require('sequelize')
+  const sequelize = new Sequelize('postgres://fakeUser:pass@fake.com:1111/fake')
+  const SEQUELIZE_JSON_TABLE = [
+    [
+      'string',
+      Sequelize.STRING,
+      { type: 'string' }
+    ],
+    [
+      'string',
+      Sequelize.CHAR,
+      { type: 'string' }
+    ],
+    [
+      'string 123',
+      Sequelize.STRING(1234),
+      {
+        type: 'string',
+        maxLength: 1234
```

Comment:
Does `maxLength` refer to character length or byte length in JSON schema? And similarly does `Sequelize.STRING(1234)` refer to max character length or byte length? We shouldn't conflate the two.

Reply:
json's maxLength refers to max char length.
sequelize's length refers to char length. There is a boolean option for binary, but it doesnt seem to affect the length when its binary=true https://github.com/sequelize/sequelize/blob/master/lib/data-types.js#L62.